src/api/data-migration/version-1-to-version-2/create-new-application-relationships/index.ts

```diff
@@ -29,20 +29,19 @@ const createNewApplicationRelationships = async (connection: Connection) => {
     const applications = await getAllApplications(connection);
 
     const newRelationships = await Promise.all([
```

Comment:
do you need a Promise.all if they're all awaits?

Reply:
No, good spot, sorry added this whilst debugging. Removed